### DISCUSSION TITLE
Regarding whether the Swerve Drive part of the SOP is planned to be supplemented

### DISCUSSION THREAD

**Discussion by smilelsb** [Choate-Robotics/7407-RobotPy-Toolkit#14] [General]
感谢7407团队开源的代码，在这过程中学习到很多。看完团队撰写的SOP文件，也对我们的程序框架有所理解。
我们今年第一次使用Swerve Drive（SDS MK4），而且我们一直使用的是Robopy，现在关于这部分的资料太少了，有幸看到这份代码。
不知道团队是否会更新SOP中的“Swerve Drive”章节，我们团队也想学习学习。十分感谢。

**Comment by mbardoe** [comment]
So glad that you found it useful. We will update the SOP for Swerve Drive soon, and there are other good python teams that have their code on GitHub. You might want to look at the Wolverines https://github.com/1757WestwoodRobotics and Ctrl-Z https://github.com/CtrlZ-FRC4096

**Reply by smilelsb** [reply]
Thank you very much for your reply. I am also looking forward to the updated SOP of Swerve Drive.
It is a coincidence that the github pages of your team and 1757 and 4096 have always been on my browser. Through comparison before, I found that our 7407 code is more suitable for us.
Looking forward to the update and thanks again for your reply.
